app/models/host_aspect_base.rb

```diff
@@ -0,0 +1,25 @@
+class HostAspectBase < ActiveRecord::Base
```

Comment:
Nm, read the explanation. Might make sense to rename `HostAspect`, as it's causing confusion.